CSharp/BatchExplorer/ViewModels/ResizePoolViewModel.cs

```diff
@@ -90,14 +132,18 @@ static ResizePoolViewModel()
             deallocationOptionValues.AddRange(Enum.GetNames(typeof (ComputeNodeDeallocationOption)));
         }
 
-        public ResizePoolViewModel(IDataProvider batchService, string poolId, int? currentDedicated)
+        public ResizePoolViewModel(IDataProvider batchService, string poolId, int? currentDedicated, string currentAutoScaleFormula)
         {
             this.batchService = batchService;
 
             this.PoolId = poolId;
             this.TargetDedicated = currentDedicated ?? 0;
-            this.DeallocationOptionString = null;
-
+            this.DeallocationOptionString = null;
+            if (string.IsNullOrEmpty(currentAutoScaleFormula))
+                this.IsFixed = true;

```

Comment:
Feels like IsFixed and IsAutoScale should be read-only properties that are derived on demand from the autoscale formula (with suitable INPC shenanigans). This would avoid needing to manually keep them in sync.